packages/svelte-kit/lib/svelte-kit-routing-provider.ts

```diff
@@ -0,0 +1,35 @@
+import type { RouteChangeSpan, RoutingProvider, StartRouteChangeCallback } from '@bugsnag/browser-performance'
+import { onSettle } from '@bugsnag/browser-performance'
+import type { AfterNavigate, BeforeNavigate } from '@sveltejs/kit'
+
+type NavigateCallback<T> = (callback: (navigation: T) => void) => void
+
+export class SvelteKitRoutingProvider implements RoutingProvider {
+  constructor (private beforeNavigate: NavigateCallback<BeforeNavigate>, private afterNavigate: NavigateCallback<AfterNavigate>, private base = window.location.origin) {}
+
+  // SvelteKit exposes the route as a URL in the beforeNavigate hook
+  // so we can use it directly by converting to a string.
+  resolveRoute (url: URL) {
+    return url.toString()
+  }
```

Comment:
I'm a bit confused by this. I think the route change plugin expects `resolveRoute` to convert an absolute url to a route. 

When the plugin is initialised, it calls `resolveRoute` with the current URL (`window.location.href`) to get the initial route. So I think this would set that as the full URL, which I don't think is what we want.

It also uses the URL to set the `bugsnag.browser.page.url` attribute on the span - so the `bugsnag.browser.page.route` and `bugsnag.browser.page.url` attributes would always be the same?